packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -337,6 +347,9 @@ export class GarbageCollector implements IGarbageCollector {
 
     // Keeps track of the GC state from the last run.
     private previousGCDataFromLastRun: IGarbageCollectionData | undefined;
+    // Keeps track of the serialized GC blob from the last run.
+    private latestSerializedGCBlobs: Map<string, string> = new Map();
```

Comment:
These should have the prefix `latestSummary` indicating that this is the state from the latest summary.
Also, they should be called `latestSummaryState` and `pendingSummaryState`. serializedGCBlobs doesn't really give any info about what they are.